### PR TITLE
add flag for nats connection name

### PR DESCRIPTION
**Pull request by jordan-rash**

### CODE REVIEW

File: nex/config.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (kthomas) [inline nex/config.json:1]: Did you mean to commit this?

File: nex/nex.go [hunk 2]
```diff
@@ -71,6 +71,7 @@ func init() {
 	ncli.Flag("logjson", "Log JSON").Default("false").Envar("NEX_LOGJSON").UnNegatableBoolVar(&Opts.LogJSON)
 	ncli.Flag("context", "Configuration context").Envar("NATS_CONTEXT").PlaceHolder("NAME").StringVar(&Opts.ConfigurationContext)
 	ncli.Flag("no-context", "Disable NATS context discovery").UnNegatableBoolVar(&Opts.SkipContexts)
+	ncli.Flag("conn-name", "Name of NATS connection").StringVar(&Opts.ConnectionName)
```
Comment (autodidaddict) [inline nex/nex.go:74]: If this flag is left off, should we use a reasonable default, like `nex-vXX` so the default behavior is to identify nex and its version?

File: nex/config.json (hunk 1 above)
Comment (autodidaddict) [inline nex/config.json:1]: Are we planning on checking this file in, or is this an artifact of running nex?

**Review by autodidaddict — CHANGES_REQUESTED** [review]

File: nex/config.json (hunk 1 above)
Comment (jordan-rash) [reply nex/config.json:1]: YESSSSS!!!!....no

**Review by kthomas — APPROVED** [review]

File: nex/nex.go (hunk 2 above)
Comment (jordan-rash) [reply nex/nex.go]: done!

**Pull request merged**